playbooks/roles/setup-caasp-workers/tasks/osh-deployer-in-hosts.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Get the facts from the deployer node from the worker nodes
+  setup:
+  delegate_to: "{{ item }}"
+  delegate_facts: True
+  loop: "{{ groups['osh-deployer'] }}"
+
+- name: Configure the hosts for local registry if developer mode is enabled
```

Comment:
Where's the check for `developer_mode` ?

Reply:
ditto.